clash-prelude/src/Clash/Explicit/Signal.hs

```diff
@@ -367,7 +367,7 @@ systemResetGen = resetGen
 --
 -- __NB__: This function is /not/ synthesizable
 freqCalc :: Double -> Integer
-freqCalc = toInteger . hzToPeriod
+freqCalc = toInteger . hzToPeriod . realToFrac
```

Comment:
You can just remove this function since we cannot backport this PR to the 1.2 branch.

Reply:
I'll do it tomorrow!

Reply:
Done